package-drivers.bash

```diff
@@ -1,52 +1,54 @@
 #!/bin/bash
+set -e
 
-PWD=$(pwd)
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
To avoid problems with symlinks and relative paths, use `dirname $(realpath $0)` instead.

Reply:
Oh interesting, I've been using `BASH_SOURCE` for 5-6 years and never run into any issues. What problems crop up with symlinks?

Reply:
There are several SO answers that mention symlinks being a problem for this one-liner but I can't find a source. See https://stackoverflow.com/questions/421772/how-can-a-bash-script-know-the-directory-it-is-installed-in-when-it-is-sourced-w as an example. `realpath` and `readlink -f` will both resolve symlinks correctly.